server/repository/tagRepository.js

```diff
@@ -0,0 +1,47 @@
+import Model from '../models';
+
+const { Tags, ArticleTag } = Model;
+
+/**
+ * Tag repository class
+ */
+class TagRepository {
+  /**
+ * Function to create a new tag
+ * @param {string} tagName
+ * @param {number} articleId
+ * @returns {object} tag
+ */
+  static async createTag(tagName, articleId) {
+    const tag = await Tags.findOrCreate({
+      where: {
+        tagName,
```

Comment:
Can we handle tagnames that are not submitted as lowercase?
So we do not have duplicated tag names with different cases.

Reply:
Nice catch @TheoOkafor I will see to this.

Reply:
Fixed